### DISCUSSION TITLE
[Question] where does youtube-dlc look for youtube-dl.conf

### DISCUSSION THREAD

**Discussion by sdsucks** [blackjack4494/youtube-dlc#194] [Q&A]
## Checklist

- [x] I'm asking a question
- [x] I've looked through the README and FAQ for similar questions
- [x] I've searched the bugtracker for similar questions including closed ones

## Question

youtube-dlc doesn't apply those options in c:\Users\MyName\youtube-dl.conf.
manually adding the options in the command line works just fine, of course.

**Comment by blackjack4494** [comment]
If you don't specify `--config-location` to use your `youtube-dl.conf` it will use `youtube-dlc.conf` by default.

**Comment by NextDev65** [comment]
![image](https://user-images.githubusercontent.com/12612637/101068624-90f36400-355e-11eb-897b-b62a5ab678c8.png)
btw on windows by %APPDATA% everyone means AppData/Roaming
what I do is make the 2 folders (because I use the original as a fallback) in %APPDATA% (which is AppData/Roaming)
![image](https://user-images.githubusercontent.com/12612637/101068683-a23c7080-355e-11eb-89af-811d4625cf7a.png)
and in youtube-dl I have youtube-dl.conf and in youtube-dlc I have youtube-dlc.conf, a symbolic link to youtube-dl.conf.
symlink are easy to do in commandline, just type mklink symlinkfilepath originalfilepath
